### PR TITLE
feat: business-oriented introduction

### PR DESCRIPTION
**Pull request by nissimsan**
This adds more to the introduction, aiming to explain to business folks how the technologies we work with solves supply chain problems.

### CODE REVIEW

File: docs/sections/abstract.html
Comment (TallTed) [inline docs/sections/abstract.html:8]: For as long as I've been involved, it has been asserted that this traceability vocab is about more than _supply chain_ traceability.  This edit adjusts the abstract to allow for such broader applicability (though it doesn't delve deeply into that breadth).  Similar edits will be needed in the larger section above, if this abstract edit is acceptable.
```suggestion
    Verifiable Credentials related to supply chain and other traceability 
    information, similar to what is often referred to as "provenance",
    including country of origin, chemical properties, mechanical properties,
    and other attributes of products and materials.  VCs using these terms 
    can then be used to help determine the legitimacy of organizations
    participating in global trade and the status of the products and 
    materials described therein.
```

Comment (nissimsan) [reply docs/sections/abstract.html:8]: Excellent, @TallTed !

**Comment by TallTed** [comment]
I don't think this should be merged until it addresses [the comment I made with my suggested changes to the abstract](https://github.com/w3c-ccg/traceability-vocab/pull/310#discussion_r811320899), which I quote here where it won't be hidden --

> For as long as I've been involved, it has been asserted that this traceability vocab is about more than supply chain traceability. [This edit](https://github.com/w3c-ccg/traceability-vocab/pull/310#discussion_r811320899) adjusts the abstract to allow for such broader applicability (though it doesn't delve deeply into that breadth). Similar edits will be needed in the larger section above, if this abstract edit is acceptable.

**Review by mkhraisha — APPROVED** [review]

**Review by TallTed — CHANGES_REQUESTED** [review]
Repeating my earlier comment, as a change request that may be more obvious as a "don't merge yet!"
-------
I don't think this should be merged until it addresses https://github.com/w3c-ccg/traceability-vocab/pull/310#discussion_r811320899, which I quote here where it won't be hidden --

> For as long as I've been involved, it has been asserted that this traceability vocab is about more than supply chain traceability. https://github.com/w3c-ccg/traceability-vocab/pull/310#discussion_r811320899 adjusts the abstract to allow for such broader applicability (though it doesn't delve deeply into that breadth). Similar edits will be needed in the larger section above, if this abstract edit is acceptable.

**Comment by nissimsan** [comment]
@TallTed , kind poke - you wanted to do some rewording still? Alternatively we can just merge and reword after.

**Comment by TallTed** [comment]
I'm concerned that rewording after merge is much less likely to happen than rewording before (delayed) merge.  I have not been able to get to this yet; it is still my intention to do so, soon.

**Comment by nissimsan** [comment]
@TallTed, I just read through my PR with the eyes of your comment. 

Your addition to the Abstract-section is included in the PR, adding the aspect of provenance (excellent point!). 

The intention of the remaining parts of the PR to explain to newcomers/business folks/anyone how the technology-mix we are applying (VC+LD+DID) is revolutionary and digitally enabling. I picked somewhat traditional supply chain use case examples, aiming to explain these abstract new technology by applying them to business problems which the audience can relate to. 

I agree that including the provenance aspect among supply chain challenges will be good. I would suggest adding a paragraph about it under Challenges. 

Provenance is also an excellent application of DID and especially VC, and paragraphs could be added here too with appropriate examples. These aren't absolutely necessary for my main ambition (to explain the technology), but it will  broaden the audience and applicability. Please don't overdo it either, though: these sections are not supposed to be flushing out use cases (that's what the next section of the document is for).

I don't see any issues in adding these elements in a subsequent PR (except for the human element of never getting around to it :) ).

**Review by BenjaminMoe — APPROVED** [review]

**Comment by nissimsan** [comment]
@TallTed, not seeing any changes on this, so I hope you're okay with merging today and get your changes in separately. Like I said Wednesday ^ I'm in full support of refinements, but it's very hard to progress with blocked PRs. Hope you understand!

**Review by mprorock — APPROVED** [review]

File: docs/index.html
Comment (TallTed) [inline docs/index.html:288]:
```suggestion
        results in effort and IT costs for both the sender and receiver to ensure this common understanding.
```

File: docs/index.html
Comment (TallTed) [inline docs/index.html:292]:
```suggestion
        Linked Data explicitly ties every term used to a precise Uniform Resource Identifier (URI). Much like a URL pointing
```

File: docs/index.html
Comment (TallTed) [inline docs/index.html:294]:
```suggestion
        there is no interpretation to be done, as the receiving computer can automatically understand it. Beyond the obvious
```

File: docs/index.html
Comment (TallTed) [inline docs/index.html:299]:
```suggestion
        Linked Data is not new; it is what drives internet indexing and search engines. This traceability-vocab specification
```

File: docs/index.html
Comment (TallTed) [inline docs/index.html:311]:
```suggestion
        different types of DIDs exist, serving different requirements from a one-off usage to an enterprise-security grade
```

File: docs/index.html
Comment (TallTed) [inline docs/index.html:313]:
```suggestion
        organization long-term representation. In itself, a DID is trustless, meaning that anyone can make and control a DID at any
        time. The value of a DID comes from the credentials tied to it, typically by other organizations and/or governments.
```

File: docs/index.html
Comment (TallTed) [inline docs/index.html:322]:
```suggestion
        identification can also be used, but use of DIDs is encouraged to ensure that actual control of an identity can be proven.
```

File: docs/index.html
Comment (TallTed) [inline docs/index.html:328]:
```suggestion
        This section conceptualizes the introduced emerging technology standards &mdash; Verifiable Credentials, Linked Data, and
        Decentralized Identifiers &mdash; with a real-world supply chain use case.
```

File: docs/index.html
Comment (TallTed) [inline docs/index.html:333]:
```suggestion
        The Commercial Invoice is a critical supply chain document, essential to customs, where it is used for duty
        determination, among others. This document includes the description of the goods, where the goods are being shipped from and
        to, and a value of the goods, just to name a few of the data points. This document is supplied by the shipper.
```

File: docs/index.html
Comment (TallTed) [inline docs/index.html:336]:
```suggestion
        The Commercial Invoice document is typically exchanged via PDF, email, or EDI. With the use of Verifiable Credential
```

File: docs/index.html
Comment (TallTed) [inline docs/index.html:337]:
```suggestion
        technology, we are able to digitize a Commercial Invoice into a verifiable Commercial Invoice credential. Each
```

File: docs/index.html
Comment (TallTed) [inline docs/index.html:339]:
```suggestion
        data point on the Commercial Invoice VC is mapped to a semantic model for common definition using Linked Data.
        For example, the “Consignee” field or the “Shipper” field on the Commercial Invoice are defined
```

File: docs/index.html
Comment (TallTed) [inline docs/index.html:347]:
```suggestion
        In addition, certain data points on the Commercial Invoice are used to identify an organization. For example,
        the “Consignee” field identifies who the receiver of the goods is, and the “Shipper” field identifies who the
        shipper of the goods is. Each “who” in our example is an organization. Using a Decentralized Identifier (DID),
        for example, the consignee organization can self-authenticate as the receiver of the goods.
```

File: docs/index.html
Comment (TallTed) [inline docs/index.html:349]:
```suggestion
        The Shipper’s Decentralized Identifier is included as the Verifiable Credential issuer, binding the claims on the
```

File: docs/index.html
Comment (TallTed) [inline docs/index.html:351]:
```suggestion
        the Shipper to the Consignee; anyone to whom the Commercial Invoice is presented can verify that it originates from the
```

File: docs/index.html
Comment (TallTed) [inline docs/index.html:352]:
```suggestion
        Shipper and is targeted to the Consignee.
```

File: docs/index.html
Comment (TallTed) [inline docs/index.html:355]:
```suggestion
        This example portrays a specific use case with a Commercial Invoice and shows how Verifiable Credentials, Linked Data, and
```

**Review by TallTed — CHANGES_REQUESTED** [review]

**Comment by TallTed** [comment]
Careful what you wish for?

**Comment by nissimsan** [comment]
Alright, done (github not the ideal text editor). :D 
Cheers @TallTed !

**Pull request merged**
